docusaurus/docs/guides/kubernetes/workload-tunneling/index.md

```diff
@@ -38,34 +39,38 @@ In the chart above, there are several strategies and solutions for connecting a
 
-#### [Loopback Proxy Sidecar](./kubernetes-sidecar.md)
+#### [ziti-tunnel as Loopback Proxy Sidecar](./kubernetes-sidecar.md)
 
 You can deploy a loopback proxy sidecar for pod egress to OpenZiti Services. The sidecar binds OpenZiti Services to a local port on the pod's loopback interface, e.g., 127.0.0.1:8443. The workload application must be configured to connect to the local port. This mode of operation has a few differences in comparison to the [transparent proxy sidecar](./kubernetes-sidecar.md) quickstart.
 
 1. The sidecar container runs `ziti tunnel proxy` instead of `ziti tunnel tproxy`.
 2. The sidecar container does not need the CAP_NET_ADMIN capability.
 3. The sidecar container does not need a `dnsPolicy` or explicit nameserver.
 
+#### [ziti-router as Cluster Service Proxy](/guides/kubernetes/hosting/kubernetes-router.mdx)
+
+The OpenZiti Router can be deployed to provide a cluster-wide proxy for OpenZiti Services, optionally exposing the Ziti Services' proxy ports with an Ingress or LoadBalancer. This is accomplished by first creating the router with tunnel mode enabled, then deploying the router Helm chart with input values specifying each Ziti Service for which the router's tunnel identity is authorized by a Dial Service Policy. For more information, see the [OpenZiti Router](/guides/kubernetes/hosting/kubernetes-router.mdx) page. 
+
 ### Ingress to Cluster Services
 
 Any OpenZiti Tunneler can be used to "host" an OpenZiti Service. This hosting begins a few moments after an OpenZiti Service becomes authorized for the OpenZiti Identity in use by the SDK. This section is about different ways to deploy an OpenZiti Tunneler to provide ingress to cluster services.
 
-#### [OpenZiti Router](/guides/kubernetes/hosting/kubernetes-router.mdx)
+#### [ziti-router as Reverse Proxy](/guides/kubernetes/hosting/kubernetes-router.mdx)
 
-The OpenZiti Router's built-in tunneler can provide ingress to cluster services. This is done by creating the router with tunneler mode enabled, and then installing the OpenZiti Router with Helm with the tunneler mode set to `host` (the default). For more information, see the [OpenZiti Router](/guides/kubernetes/hosting/kubernetes-router.mdx) page.
+The OpenZiti Router's built-in tunneler can provide ingress to cluster services by acting as a reverse proxy for traffic arriving at the router via Ziti and then flowing out to a normal cluster service. This is accomplished by creating the router with tunneler mode enabled, and then installing the OpenZiti Router Helm chart with the tunneler mode set to `host` (the default). Each "hosted" Ziti service must have a Ziti hosting configuration that specifies the server address that is reachable by the router pod, e.g., the cluster service's domain name. For more information, see the [OpenZiti Router](/guides/kubernetes/hosting/kubernetes-router.mdx) page.
```

Comment:
Can you remove words here? There are a lot of new words that don't feel like they add extra clarity, and instead possibly add confusion to me. The addition of discussing the 'hosted' service need a 'hosting configuration' seems redundant and not actually relevant to this section of doc.